README.md

```diff
@@ -36,12 +36,20 @@ yarn install
 ```
 
 ### Compiles and hot-reloads for development
-This command is used to run the application in a development environment via localhost.
+This command is used to run the frontend application only in a development environment via localhost.
 
 ```
 yarn serve
 ```
 
+To run the backend and the frontend application simultaneously, run 
+
+```
+npm run dev
```

Comment:
Since we're using yarn for this project, change this to `yarn dev` instead. yarn and npm are interchangeable btw

Reply:
Done!